cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js

```diff
@@ -0,0 +1,13 @@
+import { Button, Modal } from '../../../../../../interactors';
+
+const exceptionDeleteModal = Modal({ id: 'delete-matchProfiles-exception-modal' });
+
+export default {
+  closeException: () => cy.do(exceptionDeleteModal.find(Button('Close')).click()),
+  verifyExceptionMessage: () => cy.expect(
+    exceptionDeleteModal.has({
+      message:
+          'This match profile cannot be deleted, as it is in use by one or more job profiles.',
```

Comment:
The exception message in the test case is different, please clarify this part with the functional QA engineer

Reply:
could you please point where you can see exception message in the test case?

Reply:
sure, in the step 3
You will be presented with a "Cannot delete match profile" prompt and your only option will be to click "Close" and return to the Match profile view